improve docs about setting up hcp operator (#847)
hashicorp/terraform-docs-common: website/docs/cloud-docs/integrations/kubernetes/setup.mdx

   kubectl -n $NAMESPACE create secret generic workspacesecrets --from-literal=secret_key=abc123
   ```

1.  Add the HashiCorp Helm repository.

>     ```

Reviewer: I'm so sorry to be annoying, and I know this wasn't in the original PR, but would you mind adding a language to all the codeblocks in the file? I believe it'd be `shell-session` for all of them, and it'll really help the syntax of the TFE commands pop.

    helm repo add hashicorp https://helm.releases.hashicorp.com
    ```

> 1.  Install the [HCP Terraform Operator for Kubernetes with Helm](https://github.com/hashicorp/hcp-terraform-operator).  By default, the operator communicates with `app.terraform.io`. When deploying in a self-managed Terraform Enterprise, it is required to set the `operator.tfeAddress` to the specific hostname of the Terraform Enterprise instance.

Reviewer: ```suggestion
1.  Install the [HCP Terraform Operator for Kubernetes with Helm](https://github.com/hashicorp/hcp-terraform-operator).  By default, the operator communicates with `app.terraform.io`. When deploying in a self-managed Terraform Enterprise instance, you must set the `operator.tfeAddress` to the specific hostname of your instance.
```

>     - For HCP Terraform Cloud (default)

Reviewer: We don't use "Terraform Cloud" anymore:
```suggestion
    - For HCP Terraform (default)
```

    
    ```
    helm install --namespace ${RELEASE_NAMESPACE} hashicorp/hcp-terraform-operator tfc-operator 
    ```

>     - For self-managed Terraform Enterprise, specify your instance's hostname

Reviewer: ```suggestion
    - For self-managed Terraform Enterprise, you must specify your instance's hostname.
```

    ```
    helm install --namespace ${RELEASE_NAMESPACE} hashicorp/hcp-terraform-operator tfc-operator \
      --set operator.tfeAddress="TERRAFORM_ENTERPRISE_HOSTNAME"
    ```

>     - Alternatively, you can set this configuration in the [value.yaml](https://github.com/hashicorp/hcp-terraform-operator/blob/main/charts/hcp-terraform-operator/values.yaml) file.

Reviewer: ```suggestion
    - Alternatively, you can set up your Terraform Enterprise hostname in the [value.yaml](https://github.com/hashicorp/hcp-terraform-operator/blob/main/charts/hcp-terraform-operator/values.yaml) file.
```

    ```yaml
    operator:
      tfeAddress: <TERRAFORM_ENTERPRISE_HOSTNAME>
    ```

>     - Then, apply the configuration with.

Reviewer: ```suggestion
    - After setting your up configuration, apply it with the following command.
```

    ```
    helm install --namespace ${NAMESPACE} hashicorp/hcp-terraform-operator tfc-operator -f value.yaml
    ```

> 1.  To create a Terraform workspace, agent pool or etc, you can find different [examples](https://github.com/hashicorp/hcp-terraform-operator/tree/main/docs/examples) of the YAML manifests.

Reviewer: ```suggestion
1. For examples of features that the HCP Terraform Operator for Kubernetes with Helm supports, refer to our [example YAML manifests](https://github.com/hashicorp/hcp-terraform-operator/tree/main/docs/examples).
```
